.github/workflows/check-pkg-availability.yml

```diff
@@ -0,0 +1,129 @@
+on:
+  workflow_call:
+    inputs:
+      module:
+        type: string
+        description: The module name
+        required: true
+      major_version:
+        type: string
+        description: The version
+        required: true
+    secrets:
+      registry_username:
+        required: true
+      registry_password:
+        required: true
+
+jobs:
+  check-stable-packages:
+    name: check-stable-packages ${{ matrix.distrib }}
+    runs-on: [ubuntu-22.04]
+    container:
+      image: ${{ vars.DOCKER_PROXY_REGISTRY_URL }}/${{ matrix.image }}
+      credentials:
+        username: ${{ secrets.registry_username }}
+        password: ${{ secrets.registry_password }}
+    strategy:
+      matrix:
+        include:
+        - image: 'almalinux:8'
+          distrib: el8
+        - image: 'almalinux:9'
+          distrib: el9
+        - image: 'debian:bullseye'
+          distrib: bullseye
+
+    steps:
+      - name: Checking stable package availability
+        run: |
+          set -eux
+
+          # Set major version
+          MAJOR_VERSION=${{ inputs.major_version }}
+          COMBINATION="${{ matrix.image }}:$MAJOR_VERSION"
+
+          # Set php version
+          PHP_REPO=""
+          case "$MAJOR_VERSION" in
+            23.10 | 23.04 | 22.10)
+              PHP_REPO="remi-8.1"
```

Comment:
I don't think we use the remi repo for 23.10 but to be confirmed by @kduret

Reply:
since official documentation states this 
`dnf install -y dnf-plugins-core
dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
dnf install -y https://rpms.remirepo.net/enterprise/remi-release-8.rpm
dnf config-manager --set-enabled 'powertools'`
I have to include it, but if it is indeed not needed anymore, I can remove it (and it should not appear in doc either)